packages/cli/src/eventbus/event.types.ts

```diff
@@ -257,10 +257,6 @@ export type Event = {
 		success: boolean;
 	};
 
-	'security-audit-generated-via-cli': {
```

Comment:
does it make sense to revert [this](https://github.com/n8n-io/n8n/pull/10099/files#diff-9714e1d36a8b4bba619dd05e0ed98de04b29d9370ef55058b37fb953b6b8f578L7) change then?

Reply:
Amazing catch